native/src/modules/layout/containers/PermissionSnackbarContainer.js

```diff
@@ -53,7 +53,7 @@ class PermissionSnackbarContainer extends React.Component<PropsType, StateType>
     const pushNotificationStatus = await checkPushNotificationPermission()
 
     const showLocationSnackbar = settings && settings.proposeNearbyCities === true &&
-      [RESULTS.BLOCKED, RESULTS.DENIED].includes(locationStatus) && this.landingRoute()
+      [RESULTS.BLOCKED, RESULTS.DENIED, RESULTS.UNAVAILABLE].includes(locationStatus) && this.landingRoute()
```

Comment:
Not sure if we want this. What should the user do about it? There is nothing the user can do about it as this is because
* the feature is not available at all on the device
* or we restrict it
I don't think it really matters, so good to go!

Reply:
I would not show it for the unavailable state. So I would leave it as it is.